portfolio/src/main/webapp/index.html

```diff
@@ -104,8 +104,14 @@ <h3><u>Techincal Skills</u></h3>
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomGreeting()">FACT</button>
       <div id="greeting-container"></div>
-    </div>
-    <p>Below is me getting json files from a server and outputting the data</p>
+    <form action="/data" method="POST">
```

Comment:
I think we want to do this asynchronously via JavaScript. This will cause a post request to the server and your Java servlet is then redirecting. Is this what we really want (the form POST)? I'll have to have a look at the walkthrough for this again.